tests/e2e/tests/critical-path/browser/bulk-delete.e2e.ts

```diff
@@ -0,0 +1,191 @@
+import { KeyTypesTexts, rte } from '../../../helpers/constants';
+import { acceptLicenseTermsAndAddDatabaseApi } from '../../../helpers/database';
+import { BrowserPage, BulkActionsPage, MyRedisDatabasePage } from '../../../pageObjects';
+import { commonUrl, ossStandaloneRedisearch } from '../../../helpers/conf';
+import { deleteStandaloneDatabaseApi } from '../../../helpers/api/api-database';
+import { Common } from '../../../helpers/common';
+import { addHashKeyApi, addSetKeyApi } from '../../../helpers/api/api-keys';
+import { deleteAllKeysFromDB, populateDBWithHashes } from '../../../helpers/keys';
+
+const browserPage = new BrowserPage();
+const bulkActionsPage = new BulkActionsPage();
+const common = new Common();
+const myRedisDatabasePage = new MyRedisDatabasePage();
+
+const keyNames = [common.generateWord(20), common.generateWord(20)];
+const hashKeyParameters = { keyName: keyNames[0], fields: [{ field: common.generateWord(20), value: common.generateWord(20) }] };
+const setKeyParameters = { keyName: keyNames[1], members: [common.generateWord(20)] };
+const dbParameters = { host: ossStandaloneRedisearch.host, port: ossStandaloneRedisearch.port };
+
+fixture `Bulk Delete`
+    .meta({ type: 'critical_path', rte: rte.standalone })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneRedisearch, ossStandaloneRedisearch.databaseName);
+        await addHashKeyApi(hashKeyParameters, ossStandaloneRedisearch);
+        await addSetKeyApi(setKeyParameters, ossStandaloneRedisearch);
+    })
+    .afterEach(async() => {
+        // Clear and delete database
+        await deleteAllKeysFromDB(dbParameters.host, dbParameters.port);
+        await deleteStandaloneDatabaseApi(ossStandaloneRedisearch);
+    });
+test('Verify that user can access the bulk actions screen by clicking on the “Bulk Actions” button in the Browser', async t => {
```

Comment:
Please set a shorter name

Reply:
Hi Ofer, 
we usually match manual checklist with e2e tests (they have the same names). It allows us to find them in code easily and track in test documentation easily. 
We try creating test case names as short as possible. And when we change them in e2e code, need to change in whole test documentation. 
Specifically this test case name is one-line. Can we leave it as is please?

Reply:
I made it a bit shorter